### PR TITLE
Added modal to my page

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
added modal to my page on see project buttons click

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,6 +69,11 @@
         </div>
       </div>
     </section>
+    <section class="modal">
```
Comment (SajjadAhmad14) [inline index.html:72]: The pop-up detail for the project is missing an image. Moreover, the position of the button is not similar to the design for desktop view. Kindly see the screenshot below and solve the issues.
project1

File: index.html [hunk 2]
```diff
@@ -69,6 +69,11 @@
         </div>
```
Comment (SajjadAhmad14) [inline index.html:69]: Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and apply best practices to write professional commit messages.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Victor Chukwu, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Davidosky007** [comment]
## Hi @VICTOR-CHUKWU 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**